templates/create-backend-plugin/template.yaml

```diff
@@ -0,0 +1,108 @@
+apiVersion: scaffolder.backstage.io/v1beta3
+kind: Template
+metadata:
+  name: create-backend-plugin
+  title: Create Backend Plugin Template
```

Comment:
Add this annotation to load tech docs

```
  annotations:
    backstage.io/techdocs-ref: dir:.
```

Reply:
Thank you for this, just added the techdocs annotation!